modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -67,39 +68,48 @@ If you want to create virtual function on the default network interface, ensure
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManaged` to `true` to allow the SR-IOV Network Operator to configure and allocate a subset of virtual functions from
+a physical function that is configured externally by using the SR-IOV Network Operator. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs. test
```

Comment:
Please remove "test"

Reply:
yep I just added that to try to trigger rebuild of preview. Will remove Otherwise is it OK ?

Reply:
just removed. Thanks

Reply:
OK see LGTM now thanks @evgenLevin